projects/libs/core-library-typescript/src/state-storage/state-storage.ts

```diff
@@ -6,8 +6,11 @@ import { types } from "../common-library/types";
 import { InMemoryKVStore } from "./kvstore";
 
 import { delay } from "bluebird";
+import { STARTUP_STATUS, StartupStatus } from "../common-library/startup-status";
+import { StartupCheck } from "../common-library/startup-check";
 
-export class StateStorage {
+export class StateStorage implements StartupCheck {
+  private SERVICE_NAME = "state";
```

Comment:
same with the block storage, be more specific as we have more than one state related logic units